src/main/java/edu/byu/ece/rapidSmith/design/subsite/PropertyList.java

```diff
@@ -150,22 +141,24 @@ public boolean testPropertyValue(Object propertyKey, Object value) {
 	 * @param propertyKey the name of the property
 	 * @return the value of the specified property
 	 */
-	public Object getPropertyValue(Object propertyKey) {
+	public Object getValue(Object propertyKey) {
 		Objects.requireNonNull(propertyKey);
 
-		Property property = getProperty(propertyKey);
+		Property property = get(propertyKey);
 		return property == null ? null : property.getValue();
 	}
-	
+
+
+
 	/**
 	 * Return the given specified property as an integer. Only use this function <br>
 	 * if you are sure the property value can be safely cast to an int.
 	 * @param propertyKey the name of the property 
 	 * @return int
```

Comment:
add that null is returned if the property doesn't exist. Do we want to be returning null here, or throwing an exception? If the user is using this, then they should be pretty confident the property exists.